camel-idea-plugin/src/main/java/com/github/cameltooling/idea/runner/debugger/CamelDebuggerSession.java

```diff
@@ -393,6 +411,7 @@ private void nextStep(XSourcePosition position, boolean isOver) {
                 resume();
             }
         } else {
+//            backlogDebugger.step();
```

Comment:
commented out code can be removed.

it reveals an interesting poitn which is that the backlogDebugger;step is not used to a step action. This is somethign used in Eclipse https://github.com/jbosstools/jbosstools-fuse/blob/dd8b033accdcbb8f35d0f6c8aab5ce18e35730fc/editor/plugins/org.fusesource.ide.launcher/src/org/fusesource/ide/launcher/debug/model/CamelThread.java#L308